ext_tables.sql

```diff
@@ -7,38 +7,40 @@ CREATE TABLE tx_dlf_documents (
     tstamp int(11) DEFAULT '0' NOT NULL,
     crdate int(11) DEFAULT '0' NOT NULL,
     cruser_id int(11) DEFAULT '0' NOT NULL,
-    deleted tinyint(4) DEFAULT '0' NOT NULL,
-    hidden tinyint(4) DEFAULT '0' NOT NULL,
+    deleted smallint(6) DEFAULT '0' NOT NULL,
+    hidden smallint(6) DEFAULT '0' NOT NULL,
     starttime int(11) DEFAULT '0' NOT NULL,
     endtime int(11) DEFAULT '0' NOT NULL,
     fe_group varchar(100) DEFAULT '' NOT NULL,
-    prod_id tinytext NOT NULL,
-    location text NOT NULL,
-    record_id tinytext NOT NULL,
-    opac_id tinytext NOT NULL,
-    union_id tinytext NOT NULL,
-    urn tinytext NOT NULL,
-    purl tinytext NOT NULL,
+    prod_id varchar(255) DEFAULT '' NOT NULL,
+    location varchar(255) DEFAULT '' NOT NULL,
+    record_id varchar(255) DEFAULT '' NOT NULL,
+    opac_id varchar(255) DEFAULT '' NOT NULL,
+    union_id varchar(255) DEFAULT '' NOT NULL,
+    urn varchar(255) DEFAULT '' NOT NULL,
+    purl varchar(255) DEFAULT '' NOT NULL,
     title text NOT NULL,
     title_sorting text NOT NULL,
-    author tinytext NOT NULL,
-    year tinytext NOT NULL,
-    place tinytext NOT NULL,
-    thumbnail text NOT NULL,
-    metadata longtext NOT NULL,
-    metadata_sorting longtext NOT NULL,
+    author varchar(255) DEFAULT '' NOT NULL,
+    year varchar(255) DEFAULT '' NOT NULL,
+    place varchar(255) DEFAULT '' NOT NULL,
+    thumbnail varchar(255) DEFAULT '' NOT NULL,
+    metadata text NOT NULL,
+    metadata_sorting text NOT NULL,
     structure int(11) DEFAULT '0' NOT NULL,
     partof int(11) DEFAULT '0' NOT NULL,
-    volume tinytext NOT NULL,
-    volume_sorting tinytext NOT NULL,
+    volume varchar(255) DEFAULT '' NOT NULL,
+    volume_sorting varchar(255) DEFAULT '' NOT NULL,
     collections int(11) DEFAULT '0' NOT NULL,
     owner int(11) DEFAULT '0' NOT NULL,
     solrcore int(11) DEFAULT '0' NOT NULL,
-    status tinyint(4) unsigned DEFAULT '0' NOT NULL,
+    status smallint(6) unsigned DEFAULT '0' NOT NULL,
 
     PRIMARY KEY (uid),
-    KEY parent (pid),
-    INDEX partof (partof)
+    KEY (pid),
```

Comment:
Actually this doesn't work at all with TYPO3 and Doctrine. The key needs an index name otherwise installation via extension manager is not possible.

See error message in TYPO3 8.7 (7.6 shoud be similar):

In SchemaException.php line 75:
                                                                         
  An index with name '' was already defined on table 'tx_dlf_documents'.

Reply:
I added index names according to #360 and enhanced the `language` index by adding `l18n_parent` as you did in #360 
All indexes should now be the same as in `master` which allows a smooth upgrade.